tests/e2e/hakukohde.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect } from '@playwright/test';
+import {
+  expectAllSpinnersHidden,
+  expectPageAccessibilityOk,
+} from './playwright-utils';
+import { BasicTab } from '@/app/haku/[oid]/hakukohde/[hakukohde]/hakukohde-tabs';
+
+const ToinenAsteTabs: BasicTab[] = [
+  { title: 'Hakeneet', route: 'hakeneet' },
+  { title: 'Valinnan hallinta', route: 'valinnan-hallinta' },
+  { title: 'Valintakoekutsut', route: 'valintakoekutsut' },
+  { title: 'Pistesyöttö', route: 'pistesyotto' },
+  { title: 'Harkinnanvaraiset', route: 'harkinnanvaraiset' },
+  { title: 'Hakijaryhmät', route: 'hakijaryhmat' },
+  { title: 'Valintalaskennan tulos', route: 'valintalaskennan-tulos' },
+  { title: 'Sijoittelun tulokset', route: 'sijoittelun-tulokset' },
+];
```

Comment:
Nämä voisi olla vakiona jossain constants.ts:ssä tms. kun samat oli komponentissakin käytössä.
```suggestion
const TOINEN_ASTE_TABS: BasicTab[] = [
  { title: 'Hakeneet', route: 'hakeneet' },
  { title: 'Valinnan hallinta', route: 'valinnan-hallinta' },
  { title: 'Valintakoekutsut', route: 'valintakoekutsut' },
  { title: 'Pistesyöttö', route: 'pistesyotto' },
  { title: 'Harkinnanvaraiset', route: 'harkinnanvaraiset' },
  { title: 'Hakijaryhmät', route: 'hakijaryhmat' },
  { title: 'Valintalaskennan tulos', route: 'valintalaskennan-tulos' },
  { title: 'Sijoittelun tulokset', route: 'sijoittelun-tulokset' },
] as const;
```

Reply:
Nimesin tuon nyt uudelleen, mutta viimeistään OK-536:n yhteydessä miten nuo tabit muodostuu muuttuu paljon dynaamisemmaksi, joten mielestäni ei kannata laittaa tässäkään vaiheessa samaan paikkaan noita. Ne ei myös ole ihan yksi yhteen samat.